packages/next/build/webpack-config.ts

```diff
@@ -782,7 +784,12 @@ export default async function getBaseWebpackConfig(
       }
     },
     watchOptions: {
-      ignored: ['**/.git/**', '**/node_modules/**', '**/.next/**'],
+      ignored: [
+        '**/.git/**',
+        '**/node_modules/**',
+        '**/.next/**',
+        emacsLockfilePattern,
```

Comment:
Let's add a comment that this can be removed when this is resolved please:
https://github.com/paulmillr/chokidar/issues/955